### PR TITLE
Create Section 18 of the Controller Admin guide

### PR DESCRIPTION
**Pull request by EMcWhinn**
Added assembly and modules

Create section 18 of the Controller admin guide

https://issues.redhat.com/browse/AAP-17274

Affects `titles/controller-admin-guide`

### CODE REVIEW

File: downstream/modules/platform/con-controller-custom-logos.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[id="controller-custom-logos"]
+
+= Custom logos and images
+
+{ControllerNameStart} supports the use of a custom logo. 
+You can add a custom logo by uploading an image and supply a custom login message from the *User Interface settings* of the *Settings* menu.
```
Comment (ianf77) [inline downstream/modules/platform/con-controller-custom-logos.adoc:6]: User  Interface settings page of the

**Review by ianf77 — COMMENTED** [review]
One comment. I'll let you update

File: downstream/assemblies/platform/assembly-ag-controller-config.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+You can configure {ControllerName} settings within the *Settings* screen in the following tabs:
+
+image::ag-settings-menu-screen.png[Settings]
+
+Each tab contains fields with a *Reset* option, enabling you to revert any value entered back to the default value. 
+*Reset All* enables you to revert all the values to their factory default values.
+
+*Save* applies the changes you make, but it does not exit the edit dialog. 
+To return to the *Settings* screen, from the navigation panel select *Settings* or use the breadcrumbs at the top of the current view.
```
Comment (ianf77) [inline downstream/assemblies/platform/assembly-ag-controller-config.adoc:13]: Settings page

File: downstream/modules/platform/proc-controller-authentication.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
+[id="controller-authentication"]
+
+= Authenticating {ControllerName}
+
+Through the {ControllerName} user interface, you can set up a simplified login through various authentication types, such as GitHub, Google, LDAP, RADIUS, and SAML.
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-authentication.adoc:5]: Maybe UI?

File: downstream/modules/platform/proc-controller-configure-system.adoc [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+. From the *System* tab, choose from the following options:
+* *Miscellaneous System settings*: Enable activity streams, specify the default {ExecEnvShort}, define the base URL for the {ControllerName} host, enable {ControllerName} administration alerts, set user visibility, define analytics, specify usernames and passwords, and configure proxies.
+* *Miscellaneous Authentication settings*: Configure options associated with authentication methods (built-in or SSO), sessions (timeout, number of sessions logged in, tokens), and social authentication mapping.
+* *Logging settings*: Configure logging options based on the type you choose:
++
+image::ag-configure-aap-system-logging-types.png[Logging settings]
++
+For more information about each of the logging aggregation types, see the xref:assembly-controller-logging-aggregation[Logging and Aggregation] section.
+. Set the configurable options from the fields provided. 
+Click the tooltip image:tooltips-icon.png[Tool tip,15,15] icon next to the field that you need additional information about.
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-configure-system.adoc:26]: I can't see a tooltips-icon.png. There's question-circle.png ?

File: downstream/modules/platform/proc-controller-configure-usability-analytics.adoc [hunk 5]
```diff
@@ -0,0 +1,25 @@
+[id="controller-configure-usability-analytics"]
+
+= Configuring usability analytics and data collection
+
+Usability data collection is included with {ControllerName} to collect data to better understand how users specifically interact with it, to help enhance future releases, and to continue streamlining your user experience.
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-configure-usability-analytics.adoc:5]: to collect data to understand how users interact with it, to enhance future releases, and to streamline your user experience.

File: downstream/modules/platform/proc-controller-configure-usability-analytics.adoc [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+Only users installing a trial of {PlatformName} or a fresh installation of {ControllerName} are opted-in for this data collection.
+
+{ControllerNameStart} collects user data automatically to help improve the product. 
+You can opt out or control the way the controller collects data by setting your participation level in the *User Interface settings* in the *Settings* menu.
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-configure-usability-analytics.adoc:10]: the way {ControllerName} collects...

File: downstream/modules/platform/proc-controller-configure-usability-analytics.adoc [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+.Procedure
+
+. From the navigation panel, select menu:Settings[].
+. Select *User Interface settings* from the *User Interface* tab.
+. Click btn:[Edit].
+. Select the desired level of data collection from the *User Analytics Tracking State* drop-down list:
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-configure-usability-analytics.adoc:17]: drop-down list -> menu

File: downstream/modules/platform/proc-controller-configure-user-interface.adoc [hunk 8]
```diff
@@ -0,0 +1,11 @@
+[id="controller-configure-user-interface"]
+
+= Configuring the user interface
+
+The *User Interface* tab enables you to set {ControllerName} analytics settings, as well as configure custom logos and login messages.
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-configure-user-interface.adoc:5]: settings and configure custom...

File: downstream/modules/platform/proc-controller-obtaining-subscriptions.adoc [hunk 9]
```diff
@@ -7,18 +7,27 @@ If not, you can request a link:https://www.redhat.com/en/technologies/management
 
 .Procedure
 
+ifdef::controller-GS,controller-UG[]
 * If you already have a {PlatformName} subscription, use your Red Hat customer credentials when you launch the {ControllerName} to access your subscription information.
 See xref:controller-importing-subscriptions[Importing a subscription].
+endif::controller-GS,controller-UG[]
+ifdef::controller-AG[]
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-obtaining-subscriptions.adoc:14]: ifdef::controller-GS,controller-AG[]

File: downstream/modules/platform/proc-controller-obtaining-subscriptions.adoc [hunk 10]
```diff
@@ -7,18 +7,27 @@ If not, you can request a link:https://www.redhat.com/en/technologies/management
...
+* If you already have a {PlatformName} subscription, use your Red Hat customer credentials when you launch the {ControllerName} to access your subscription information.
+See link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html-single/automation_controller_user_guide/index#controller-importing-subscriptions[Importing a subscription].
+endif::controller-AG[]
 
 * If you have a non-Ansible Red Hat or Satellite subscription, access {ControllerName} with one of these methods:
 ** Enter your username and password on the license page.
 ** Obtain a subscriptions manifest from the link:https://access.redhat.com/management/subscription_allocations[Subscription Allocations] page on the Red Hat Customer Portal.
 ifdef::controller-GS[]
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-obtaining-subscriptions.adoc:22]: ifdef::controller-GS,controller-AG[]

File: downstream/modules/platform/proc-controller-obtaining-subscriptions.adoc [hunk 11]
```diff
@@ -7,18 +7,27 @@ If not, you can request a link:https://www.redhat.com/en/technologies/management
...
-For detailed information, see link:http://docs.ansible.com/automation-controller/4.4/html/userguide/import_license.html#obtain-sub-manifest[Obtaining a subscriptions manifest] in the Automation Controller User Guide.
+For detailed information, see link:http://docs.ansible.com/automation-controller/4.4/html/userguide/import_license.html#obtain-sub-manifest[Obtaining a subscriptions manifest] in the _{ControllerUG}_.
 endif::controller-GS[]
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-obtaining-subscriptions.adoc:24]: endif::controller-GS,controller-AG[]

File: downstream/modules/platform/proc-controller-obtaining-subscriptions.adoc [hunk 12]
```diff
@@ -29,4 +38,7 @@ endif::controller-GS[]
 ifdef::controller-UG[]
 To understand what is supported with your subscription, see xref:assembly-controller-licensing[{ControllerNameStart} licensing, updates and support].
 endif::controller-UG[]
+ifdef::controller-AG[]
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-obtaining-subscriptions.adoc:41]: ifdef::controller-GS,controller-AG[]

File: downstream/modules/platform/proc-controller-obtaining-subscriptions.adoc [hunk 13]
```diff
@@ -29,4 +38,7 @@ endif::controller-GS[]
...
+To understand what is supported with your subscription, see link:http://docs.ansible.com/automation-controller/4.4/html/userguide/license-support.html#licenses-feat-support[Red Hat Ansible Automation Platform Controller Licensing, Updates, and Support].
+endif::controller-AG[]
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-obtaining-subscriptions.adoc:43]: endif::controller-GS,controller-AG[]

**Review by ianf77 — COMMENTED** [review]
Some comments. the  best way to check the ifdefs is to build each document and compare them.

File: downstream/modules/platform/proc-controller-configure-system.adoc (hunk 4 above)
Comment (EMcWhinn) [reply downstream/modules/platform/proc-controller-configure-system.adoc:26]: I just added the tooltips-icon.png in this PR, I don't have a question-circle.png...

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**